README.md

```diff
@@ -36,6 +36,22 @@ curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/paritytec
 * Other Resources:
   * [Polkadot Wiki -> Build](https://wiki.polkadot.network/docs/build-guide)
 
+## Binaries
+
+This repo provides the source code to a number of useful binaries used throughout Polkadot SDK development. The most commonly used are:
+
+- `polkadot`: A node for the main Polkadot Relay Chain.
+- `polkadot-parachain`: A node for a generic Parachain. Can be combined with a custom chain specification to launch a custom Parachain.
+- `chain-spec-builder`: A tool to generate a custom chain specification. Can be combined with a Polkadot-SDK compatible Wasm runtime to build a custom chain specification.
+
+You can find released binaries for Linux in our [latest releases](https://github.com/paritytech/polkadot-sdk/releases/).
+
+For other platforms, you can compile the binaries yourself with:
+
+```bash
+cargo install --git https://github.com/paritytech/polkadot-sdk/ --tag polkadot-stable2407 polkadot polkadot-parachain-bin staging-chain-spec-builder
```

Comment:
I think it also needs the `polkadot-execute-worker` and `polkadot-prepare-worker` binaries to work?  
Slightly unfortunate, but we basically split the polkadot binary into three (for security reasons) 🫠

Reply:
Yes, you need those to run a node as validator.

Reply:
Where can i learn how these other 2 binaries work and what they do?

Reply:
See here for more info: https://github.com/paritytech/polkadot/pull/7337